lib/api/goal/ExecuteGoalResult.ts

```diff
@@ -64,16 +65,4 @@ export interface GoalDetails {
 /**
  * Result from goal execution
  */
-export interface ExecuteGoalResult extends GoalDetails {
-
-    /**
-     * 0 is success; non-zero exit codes will mark the goal as failed,
-     * if state is not defined
-     */
-    code: number;
-
-    /**
-     * The simple text message describing the result
-     */
-    message?: string;
-}
+export interface ExecuteGoalResult extends GoalDetails, HandlerResult { }
```

Comment:
This change is reverting a deliberate removal this inheritance. We don’t want the `ExcuteGoalResult` to extend the legacy `HandlerResult`. See 
https://github.com/atomist/sdm/issues/469 for more.